src/schemas/VIC/schema.js

```diff
@@ -17,20 +17,22 @@ let schema = {
       type: 'string',
       format: 'email'
     },
-    govtIdConfirmationCode: {
-      type: 'string',
+    photo: {
+      type: 'string'
     },
-    photoConfirmationCode: {
-      type: 'string',
-    }
+    dd214: Object.assign({}, originalDefinitions.files, {
+      minItems: 1,
+      maxItems: 1
+    })
   },
 };
 
 [
   ['fullName', 'veteranFullName'],
   ['address', 'veteranAddress'],
+  ['ssn', 'veteranSocialSecurityNumber'],
   ['phone'],
-  ['dischargeType']
+  ['date', 'dateOfBirth']
```

Comment:
we don't want XXs in the date for date of birth. also would `veteranDateOfBirth` be a better name since that's what's used in the 1990? do we want to add gender here too or are you using this schema only for the final submission?

Reply:
I was planning on having gender be one of our `view:` front end only fields. I'll update this for the other two points.